corehq/form_processor/management/commands/run_submission_reprocessing_queue.py

```diff
@@ -34,6 +35,7 @@ def handle(self, **options):
             sleep_time = 10 if num_processed < BATCH_SIZE else 0
             sleep(sleep_time)
 
+    @handle_connection_failure()
```

Comment:
can you share how does that solve the exception? or it just handles it better by closing the connection so its not retried?

Reply:
it closes the connection properly so that the next attempt won't fail with "connection already closed"

Reply:
thank you